httpsync/publisher.go

```diff
@@ -35,6 +35,10 @@ var _ http.Handler = (*publisher)(nil)
 // NewPublisher creates a new http publisher, listening on the specified
 // address.
 func NewPublisher(address string, lsys ipld.LinkSystem, peerID peer.ID, privKey ic.PrivKey) (*publisher, error) {
+	if privKey == nil {
+		return nil, errors.New("private key is nil. Can't sign head requests")
```

Comment:
```suggestion
		return nil, errors.New("private key required to sign head requests")
```
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation

I did neither of those things?